main.go

```diff
@@ -134,3 +138,18 @@ func getYear() string {
 	}
 	return strconv.Itoa(time.Now().Year())
 }
+
+func getProject() string {
+	if fProject != "" {
+		return fProject
+	}
+	dir, err := os.Getwd()
+	if err != nil {
+		return ""
+	}
+	rootDir, err := filepath.Abs(dir)
+	if err != nil {
+		return ""
+	}
```

Comment:
The os.Getwd documentation says it returns a rooted path (which, I think, means absolute path), so calling filepath.Abs on it should not be necessary. Anyway, no harm in having it.